docs/docs-content/enterprise-version/upgrade/upgrade-k8s/non-airgap.md

```diff
@@ -0,0 +1,215 @@
+---
+sidebar_label: "Non-airgap"
+title: "Upgrade Palette Installed with Kubernetes"
+description: "Learn how to upgrade self-hosted non-airgap Palette with Helm and Kubernetes."
+icon: ""
+sidebar_position: 0
+tags: ["palette", "self-hosted", "non-airgap", "kubernetes", "management", "upgrades"]
+keywords: ["self-hosted", "enterprise"]
+---
+
+This guide takes you through the process of upgrading a self-hosted Palette instance installed with Helm on Kubernetes.
+
+:::warning
+
+Before upgrading Palette to a new major version, you must first update it to the latest minor version available. Refer
+to the [Supported Upgrade Paths](../upgrade.md#supported-upgrade-paths) section for details.
+
+If your setup includes a PCG, you must also
+[allow the PCG to upgrade automatically](../../../clusters/pcg/manage-pcg/pcg-upgrade.md) before each major or minor
+Palette upgrade.
+
+:::
+
+## Prerequisites
+
+- [`kubectl`](https://kubernetes.io/docs/tasks/tools/#kubectl) and [`helm`](https://helm.sh/docs/intro/install/)
+  available in your system.
+
+- Access to the `kubeconfig` file of the target Kubernetes cluster. You must be able to interact with the cluster
+  through `kubectl` and have sufficient permissions to upgrade Palette. We recommend using a role with the cluster-admin
+  permissions.
+
+- `unzip` or a similar tool available in your system.
+
+- Access to the latest Palette Helm Chart. Refer to [Access Palette](/enterprise-version/#access-palette) for more
+  details.
+
+## Upgrade
+
+:::info
+
+Depending on your underlying infrastructure provider and Kubernetes distribution, you may need to modify these steps to
+match your environment.
+
+:::
+
+1. Open a terminal session and navigate to the directory with the Palette installation zip file. Unzip the file to a
+   **palette-install** directory.
+
+   ```shell
+   unzip release-*.zip -d palette-install
+   ```
+
+2. Navigate to the release directory inside **palette-install**.
+
+   ```shell
+   cd palette-install/charts/release-*
+   ```
+
+3. If you're using a self-hosted OCI registry, then, in a code editor of your choice, open the
+   **extras/cert-manager/values.yaml** file and replace the `controllerImage`, `webhookImage`, and `amceResolverImage`
+   image URLs with your OCI image registry URLs.
+
+   ```yaml
+   image:
+   cainjectorImage: "gcr.io/spectro-images-public/release-fips/jetstack/cert-manager-cainjector:spectro-v1.11.0-20230427"
+   // highlight-start
+   controllerImage: "<your-oci-registry-url>/spectro-images-public/release-fips/jetstack/cert-manager-controller:spectro-v1.11.0-20230427"
+   webhookImage: "<your-oci-registry-url>/spectro-images-public/release-fips/jetstack/cert-manager-webhook:spectro-v1.11.0-20230808"
```

Comment:
They don't need to update URLs in the non-airgap. They can be downloaded from internet.

Only in case of airgap they need to update it.